torch/onnx/utils.py

```diff
@@ -546,8 +547,13 @@ def _run_symbolic_function(g, n, inputs, env, operator_export_type=OperatorExpor
                 elif n.output().type().kind() == "DeviceObjType":
                     return None
                 else:
-                    raise RuntimeError("Unsupported prim::Constant kind: `{}`. Send a bug report.".format(
-                        n.kindOf("value")))
+                    is_caffe2_op = False
```

Comment:
Hm, can't it be restructured in such a way that we first run a pass that subsumes all of _caffe2 ops and only then run symbolics? This kind of check looks sketchy (or is there dead code elimination missing somewhere?

Reply:
Yeah, I think that's a good idea.